src/style.css

```diff
@@ -0,0 +1,29 @@
+* {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
don't reset your margins and paddings with * selector, it's a bad practice, you should only reset margin for body in this task